Error compiling on Debian

On Debian i have this errors
oc_feature_affine.cpp:497:94: error: ‘gen64’ was not declared in this scope
 error: ‘random_device’ is not a member of ‘std’

in member function ‘virtual void opencorr::FeatureAffine2D::compute(opencorr::POI2D*)

i use the CMakeLists.txt from examples ... i have only added add_definitions(-DOPENCV_DISABLE_EIGEN_TENSOR_SUPPORT)

--- a/src/oc_feature_affine.cpp
+++ b/src/oc_feature_affine.cpp
@@ -13,6 +13,7 @@
  */
 
 #include <numeric>
+#include <random>
 #include <omp.h>
 
 #include "oc_feature_affine.h"

